.circleci/config.yml

```diff
@@ -173,6 +173,11 @@ jobs:
     steps:
       - checkout
       - setup_nix
+      - run:
+          name: Packell test
+          command:
+            # nix-shell --pure --run 'file /nix/store/*packell*/bin/packell'
```

Comment:
remove comment?

Reply:
These are temporary tests as the CI is failing